magento/magento2
# Added missing aclResource attribute to Backend Header AdminNotification Toolbar Block

### Description (*)
Admin notification toolbar entry (bell) is visible despite admin user does not have role resource permissions to see it.

### Related Pull Requests
n/a

### Fixed Issues (if relevant)
n/a

### Manual testing scenarios (*)
1. Create Admin Role with Resource Access set to Custom
2. Edit Resources Tree so that System->Other Settings->Notifications->Show Toolbar is unchecked
3. Save Role
4. Refresh caches and re-login
Expected result: as admin user you should not see admin notifications bell icon 
Actual result: it is visible 

### Questions or comments
n/a

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#29067: Added missing aclResource attribute to Backend Header AdminNotification Toolbar Block
